src/_components/shared/footer.rb

```diff
@@ -1,10 +1,10 @@
 class Shared::Footer < Bridgetown::Component
-  def initialize(show:)
-    @show = show
+  def initialize(hide: false)
+    @hide = hide
     @site = Bridgetown::Current.site
   end
 
   def render?
-    @show
+    @hide
```

Comment:
I don't get it ? When you set `hide_footer: true` in the frontmatter, you want the footer NOT to be rendered right? So `render?` should return `false` shouldn't it?

Reply:
🤦‍♀️ This wasn't working either, thanks for pointing this out